Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -189,10 +191,13 @@ internal void RemoveObservers()
         [EditorBrowsable(EditorBrowsableState.Never), Obsolete("Use InvokeHandler<T> instead")]
         public bool InvokeHandlerNoData(int msgType)
         {
-            return InvokeHandler(msgType, null);
+            return InvokeHandlerInternal(msgType, null);
         }
 
-        public bool InvokeHandler(int msgType, NetworkReader reader)
+        [EditorBrowsable(EditorBrowsableState.Never), Obsolete("Use InvokeHandler<T> instead")]
+        public bool InvokeHandler(int msgType, NetworkReader reader) => InvokeHandlerInternal(msgType, reader);
+        // TODO: Rename this back to InvokeHandler once the obsolete above is removed
+        internal bool InvokeHandlerInternal(int msgType, NetworkReader reader)
```

Comment:
why not just make InvokeHandler internal to begin with?
don't worry about obsolete here, no one is using this from the outside anyway.

Reply:
Okay, will do.